pkg/network/network_metrics.go

```diff
@@ -0,0 +1,49 @@
+package network
+
+import (
+	"os"
+	"path/filepath"
+
+	operv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/cluster-network-operator/pkg/render"
+	"github.com/pkg/errors"
+	uns "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// RenderMultus generates the manifests of Multus
+func RenderNetworkMetricsDaemon(conf *operv1.NetworkSpec, manifestDir string) ([]*uns.Unstructured, error) {
+	if *conf.DisableMultiNetwork {
```

Comment:
rather than having `RenderNetworkMetricsDaemon` check this, you should just have this function be called as a subroutine of `RenderMultus`

Reply:
Agree, will change it.